app/proto/donations.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+package org.couchers.api.donations;
+
+import "annotations.proto";
+
+service Donations {
+  option (auth_level) = AUTH_LEVEL_SECURE;
+
+  rpc InitiateDonation(InitiateDonationReq) returns (InitiateDonationRes) {
+    // Initiate a donation
+  }
```

Comment:
Document how a client is supposed to proceed with the donation (or refer to stripe docs)